modules/installation-gcp-permissions.adoc

```diff
@@ -32,6 +32,13 @@ account requires the following permissions. If you deploy your cluster into an e
 .Required roles for creating network resources during installation
 * DNS Administrator
 
+.Required roles for installing into a shared VPC
```

Comment:
In fact, it's for `credentialsMode: Passthrough`, either installing into a shared VPC or not.